app/test_server.py

```diff
@@ -24,20 +16,75 @@ def test_true():
     assert True, "We have a problem!"
 
 
-@pytest.mark.xfail(PYTHON3 or not TRAVIS_CI, reason=REASON)
+@pytest.mark.xfail(reason='small_test() needs to be fixed.')
+def test_small_test():
+    small_test()
+
+
+@pytest.mark.xfail
 def test_invalid_url_api_call():
     response = requests.get('http://localhost:7001/api/v1/search/invalid_url')
     assert response.json()['Status Code'] == 404
+    assert response.status_code == response.json()['Status Code']
```

Comment:
how is this helping ?

Reply:
Good catch.  I will use the first form in the test above and then remove this test.